### PR TITLE
Normalize keyboard shortcuts

### PR DESCRIPTION
**Pull request by Binnette**
This PR is related to #753

Hi @yohanboniface can you please check and approuve my PR?

But please do not merge it yet, i need to "mass edit" all translations to apply the the changes.

### CODE REVIEW

**Comment by yohanboniface** [comment]
Approve, go go go :)

**Comment by Binnette** [comment]
@yohanboniface, ok it seems good to me now. Can you please review my changes?

File: umap/static/umap/locale/am_ET.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ var locale = {
     "The zoom and center have been setted.": "ዙሙ እና መሀከሉ ተወስነዋል",
     "To use if remote server doesn't allow cross domain (slower)": "ሪሞት ሰርቨሩ የዶሜይን ልውውጥን የማይፈቅድ ከሆነ ይህን ለመጠቀም (አዝጋሚ)",
     "To zoom": "ለማጉላት",
-    "Toggle edit mode (shift-click)": "Toggle edit mode (shift-click)",
+    "Toggle edit mode (Shift+Click)": "Toggle edit mode (Shift+Click)",
```
Comment (yohanboniface) [inline umap/static/umap/locale/am_ET.js]: Humm, any idea why we only have three changes in translations files, while we have more changes in the code base ?

Comment (yohanboniface) [reply umap/static/umap/locale/am_ET.js]: Ah, because only three strings have the shortcut embedded, other are on the form `L._("xxx") + " (shortcut)"`

**Pull request merged**